llvm/test/TableGen/dag-isel-regclass-emit-enum.td

```diff
@@ -27,13 +27,13 @@ def GPRAbove127 : RegisterClass<"TestTarget", [i32], 32,
 // CHECK-NEXT: OPC_CheckChild1Integer, 0,
 // CHECK-NEXT: OPC_EmitInteger32, 0|128,2/*256*/,
 // CHECK-NEXT: OPC_MorphNodeTo1None, TARGET_VAL(TargetOpcode::COPY_TO_REGCLASS),
-// CHECK-NEXT:     MVT::i32, 2/*#Ops*/, 1, 0,
+// CHECK-NEXT:     7, 2/*#Ops*/, 1, 0,
```

Comment:
A comment like `7/*MVT::i32*/, 2/*#Ops*/`?

Reply:
This is what I thought, but in VBR, it prints a number as comment in the end of VBR sequence, so if we want to also print out MVT information, it would be something like `/*MVT::i32*/ 7 /*7*/`.

Reply:
Make sense to me.